nautobot/core/navigation.py

```diff
@@ -0,0 +1,34 @@
+from nautobot.core.apps import NavMenuAddButton, NavMenuGroup, NavMenuItem, NavMenuTab
+
+
+# TODO fix initialization order issue with admin site
+_menu_items = (
```

Comment:
As noted in the PR description, setting `menu_items` here causes an order-of-initialization problem with `nautobot.core.admin`. TODO.